src/ActionGroup.php

```diff
@@ -0,0 +1,76 @@
+<?php
+declare (strict_types=1);
+
+namespace LotGD\Core;
+
+/**
+ * A grouping of navigation actions, like a submenu.
+ */
+class ActionGroup
+{
+    private $id;
+    private $title;
+    private $sortKey;
+    private $actions;
+
+    /**
+     * Create a new ActionGroup, starting with an empty set of actions.
+     * @param string $id Unique identifier for this group, in the vendor/module/group format.
+     * @param string $title Title to display to the end user. Empty string means no title.
+     * @param string $sortKey Navigation menus are displayed in the order sorted by this string.
```

Comment:
So you can control which order the menu items appear in. Say you have two groups:

Main Nav:
Village
Forest
...

Special Clan Nav:
Secret clan place
...

(or whatever, the LotGD navs often have groupings)

The sorkKey will enable modules to force an order on the groupings without having to know everything about the other groups, as would be the case if we simply used the order of the groups in the array.